src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -135,448 +130,24 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/k8s-
-    env: 
-      - name: API_SERVER_CACHE_TTL 
-        value: "1m"
-    ```
-  </Collapser>
-
-  <Collapser
-    id="kube-state-metrics-control-plane"
-    title="Specify base URLs for control plane component endpoints"
-  >
-    Use the following environment variables if any of the Kubernetes control plane components export metrics on base URLs that are different [from the defaults](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#discover-nodes-components). This is necessary for environments such as [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) when a control plane component metrics endpoint is using SSL or an alternate port.
-
-    Values of these environment variables must be base URLs of the form `[scheme]://[host]:[port]`. URLs should not include a path component. For example:
-
-    ```
-    - name: "SCHEDULER_ENDPOINT_URL"
-      value: "https://localhost:10259"
-    - name: "ETCD_ENDPOINT_URL"
-      value: "https://localhost:9979"
-    - name: "CONTROLLER_MANAGER_ENDPOINT_URL"
-      value: "https://localhost:10257"
-    - name: "API_SERVER_ENDPOINT_URL"
-      value: "https://localhost:6443"
-    ```
-
-    The `/metrics` path segment is added automatically. In addition, if the `https` scheme is used, authentication to the control plane component pod(s) is accomplished via service accounts.
-
-    <Callout variant="caution">
-      If a FQDN (fully qualified domain name) is used in a multi-master cluster, inconsistent results may be returned. Therefore, it is recommended to use `localhost` only.
-    </Callout>
-
-    <Callout variant="important">
-      Even though a custom base URL is defined for a given control plane component, the control plane component pod(s) must contain one of [the labels supported by the auto-discovery process](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#discover-nodes-components).
-    </Callout>
-
-    <Callout variant="important">
-      Even though a custom `ETCD_ENDPOINT_URL` can be defined, ETCD will always require `https` and [mTLS authentication to be configured](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#mtls-how-to).
-    </Callout>
-  </Collapser>
-</CollapserGroup>
-
-Here are some additional configurations to consider:
-
-* [Do more configuration for control plane monitoring](/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring)
-* [Link New Relic APM to the Kubernetes integration](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes)
-* [Monitor services that run on Kubernetes](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#monitor-services)
-
-## Configure the infrastructure agent [#configure-infrastructure-agent]
```

Comment:
also this is not needed anymore?

Reply:
I'm not aware of any use case where Kubernetes users wish to modify the agent settings. Can you think about one? We could make a section for "advanced settings" if needed.

Reply:
for example for the proxy https://github.com/newrelic/helm-charts/pull/448/files

I guess for fedramp endpoint, however it should be documented as well in the values.yaml

Reply:
My idea was to duplicate as few info as possible so we don't need to maintain it in two places. We can add it to the readme though!